python_modules/dagster/dagster/_core/storage/event_log/sql_event_log.py

```diff
@@ -1327,11 +1329,22 @@ def get_asset_check_summary_records(
     ) -> Mapping[AssetCheckKey, AssetCheckSummaryRecord]:
         states = {}
         for asset_check_key in asset_check_keys:
-            execution_record = self.get_asset_check_execution_history(asset_check_key, limit=1)
+            last_execution_record = self.get_asset_check_execution_history(asset_check_key, limit=1)
+            last_completed_execution_record = self.get_asset_check_execution_history(
+                asset_check_key, limit=1, status=COMPLETED_ASSET_CHECK_EXECUTION_RECORD_STATUSES
+            )
             states[asset_check_key] = AssetCheckSummaryRecord(
                 asset_check_key=asset_check_key,
-                last_check_execution_record=execution_record[0] if execution_record else None,
-                last_run_id=execution_record[0].run_id if execution_record else None,
+                last_check_execution_record=last_execution_record[0]
+                if last_execution_record
+                else None,
+                last_run_id=last_execution_record[0].run_id if last_execution_record else None,
```

Comment:
can't you get this from last_check_execution_record?

Reply:
oh this was here before lol.....

Reply:
confused, am I not getting it from last_execution_record

Reply:
yeah I just mean that it feels duplicated to also pass the run_id in when it's from an object that's also passed in.

Reply:
yea we should just add an `@property` on the object to fetch it from out of the event instead of this